Jenkinsfile.deploy

```diff
@@ -230,20 +233,49 @@ pipeline {
                 }
             }
         }
-
-    }
-    post {
-        always {
-            script {
-                def propertiesStr = deployProperties.collect{ entry ->  "${entry.key}=${entry.value}" }.join("\n")
-                writeFile(text: propertiesStr, file: 'deployment.properties')
-                archiveArtifacts(artifacts: 'deployment.properties')
+        post {
+            always {
+                script {
+                    def propertiesStr = deployProperties.collect { entry -> "${entry.key}=${entry.value}" }.join("\n")
+                    writeFile(text: propertiesStr, file: 'deployment.properties')
+                    archiveArtifacts(artifacts: 'deployment.properties')
+                }
+                cleanWs()
             }
-            cleanWs()
         }
     }
 }
 
+void updateQuickstartsVersions(){
+    maven.mvnSetVersionProperty(getOptaplannerQuickstartsMavenCommand(),"version.org.optaplanner",getProjectVersion())
+    maven.mvnVersionsUpdateParentAndChildModules(getOptaplannerQuickstartsMavenCommand(), getProjectVersion())
+    gradleVersionsUpdate(quickstartsRepository,getProjectVersion())
+    removeJbossNexusFromMavenAndGradle(quickstartsRepository)
+
+    dir(quickstartsRepository) {
+        assert !sh (script: 'grep -Rn "SNAPSHOT" --include={pom.xml,build.gradle}| grep -v -e "0.1.0-SNAPSHOT" -e ">1.0-SNAPSHOT"|cat'
+                , allowEmptyResults: true, returnStdout: true)
+        assert !sh (script: 'grep -Rn "repository.jboss.org" --include={pom.xml,build.gradle}|cat'
+                , allowEmptyResults: true, returnStdout: true)
+    }
+}
+
+void removeJbossNexusFromMavenAndGradle(String repo) {
```

Comment:
The JBoss Nexus must disappear from the `stable` branch only, as after the release:
- the `development` branch continues on with next minor snapshot version => requires the JBoss Nexus
- the release branch (e.g. 8.1.x) upgrades to the next micro snapshot version => requires the JBoss Nexus

Reply:
Yes that means I need to bring JBoss Nexus back while updating development and release version. Not the scope of this PR, but thanks for reminding

Reply:
Or maybe we remove just on the `stable` branch in a separate commit, so that we don't have to hardcode the repository in the pipeline.

Reply:
Fixed in 5144be7

Reply:
why keeping the method in here if not used ?

Reply:
See https://github.com/kiegroup/optaplanner/pull/1084#discussion_r553850802